src/Services/Config.php

```diff
@@ -90,6 +90,7 @@ protected function routes(): RoutesData
             ),
             namePrefix: $this->value(Name::Shared, 'routes.name_prefix', fallback: 'localized.'),
             redirect  : $this->value(Name::Shared, 'routes.redirect_default', fallback: false),
+            hide      : $this->value(Name::Shared, 'routes.hide_default'),
```

Comment:
You need to specify a fallback value:

```suggestion
            hide      : $this->value(Name::Shared, 'routes.hide_default', false),
```